about.html

```diff
@@ -0,0 +1,147 @@
+            celebrate glorious country heritage, culture and traditions. This
+            fest is meant to rejoice our special moments and emotions in our
+            lives with our loved ones, be it as it may, we're all country boys
+            and girls.
+          </p>
+          <p class="about__par hero__contact__text">
+            Please contact us per Email for any further questions about Honky
+            Tonk Music Festival!
+          </p>
+          <p class="about__par about__email">
+            <a href="mailto:honkytonk22@music.concert"
+              >honkytonk22@music.concert</a
+            >
+          </p>
+        </div>
+      </section>
+      <section class="logo__brand">
+        <header>
+          <h2>Honky Tonk Music Festival 2022 Logo</h2>
+          <hr />
+        </header>
+        <p class="about__par">
+          The logo of Honky Tonk Music Fest '22 was decided by a heart and soul
+          of a hillbilly lover.
+        </p>
+        <div class="logo__brand__container">
+          <img src="assets/images/logo.png" alt="honky-tonk logo" />
+        </div>
+        <hr />
+      </section>
+      <section class="past__events">
+        <header>
+          <h2>See the past Honky Tonk Musical Festivals</h2>
+        </header>
+        <hr />
+        <p class="about__par">
+          Take a look at the last two Honky Tonk Musical Festival which took
+          place in Mississippi and Texas.
+        </p>
+        <div id="past__events__list">
+          <div class="mississippi">
+            <h2>2021</h2>
+            <p>Honky-Tonk Fest '21 in Mississippi</p>
+          </div>
+          <div class="texas">
+            <h2>2020</h2>
+            <p>Honky-Tonk Fest '20 in Texas</p>
+          </div>
+        </div>
+      </section>
+      <section class="partner">
+        <header>
+          <h2>Partner</h2>
+          <hr />
+        </header>
+        <div class="partner__list">
+          <img src="assets/images/jack-daniels.png" alt="jack daniels logo" />
+          <img src="assets/images/Ford_logo_flat.svg.png" alt="ford logo" />
+          <img
+            src="assets/images/Jeff-Rubys.png"
+            alt="jeff rub's steak house logo"
+          />
+          <img src="assets/images/Budweiser-logo.png" alt="budweiser logo" />
+          <img
+            src="assets/images/south_holston.jpg"
+            alt="south holston river logo"
+          />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <img src="assets/images/logo.png" alt="logo" />
+      <p class="par" id="mobile__footer">
+        2022 Honky Tonk Music Festival. <br />
+        Some Right Reserved.
+      </p>
+      <p class="par" id="desktop__footer">
+        2022 Honky Tonk Music Festival. Some Right Reserved. <br />
+        The Honky-Tonk Musical Festival that touches the whole life of country
+        boys and girls.
+      </p>
+    </footer>
```

Comment:
- [ ]  the footer should be black in the desktop version please make sure to do that , that you stick to design